src/bp-forums/groups.php

```diff
@@ -317,6 +632,93 @@ public function edit_screen( $group = false ) {
 			}
 		}
 
+		/**
+		 * Set the selected forum in the forum dropdown from the group edit screen.
+		 *
+		 * @since BuddyBoss 1.5.7
+		 *
+		 * @param string $where        where condition forum post query.
+		 * @param object $query_object forum post query property.
+		 * 
+		 * @return string
+		 */
+		public function update_forum_dropdown_qeury( $where, $query_object ) {
```

Comment:
update_forum_dropdown_qeury here query is spelled wrong. please check it.

Reply:
Fixed https://github.com/buddyboss/buddyboss-platform/pull/2117/commits/a51489618f8f635d3ef6a0dc7d8b1a511d1be484